sortfilterproxymodel.qbs

```diff
@@ -1,12 +1,14 @@
 import qbs 1.0
+import qbs.File
+import qbs.FileInfo
 
 Product
 {
-    name: "sortfilterproxymodel"
-    type: "dynamiclibrary"
+    name: "SortFilterProxyModel"
+    type: ["dynamiclibrary", "copied qbs resources"]
```

Comment:
Oh, interesting. I didn't realize `copied qbs resources` was a type. Is this some sort of hack, or one of those hidden/undocumented things? Hopefully this doesn't change in future Qbs versions.